auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -226,8 +235,13 @@ export default class AccountInfo extends Mixins(AccountChangeMixin) {
   private async setup () {
     const accountSettings = this.getAccountFromSession()
     this.orgName = this.currentOrganization?.name || ''
-    if (this.isPremiumAccount || this.enablePaymentMethodSelectorStep) {
+    if (!this.anonAccount && this.enablePaymentMethodSelectorStep) {
       await this.syncAddress()
+      if (Object.keys(this.currentOrgAddress).length === 0) {
+        this.isCompleteAccountInfo = false
+        this.errorMessage = this.isAddressEditable ? 'This accounts profile is incomplete. You will not be able to proceed until the missing information for this account has been completed.'
```

Comment:
`This accounts profile is incomplete`, wording looks wrong to me. It could be simply `Account profile is incomplete`.